### PR TITLE
Issue 47 auth implementation (#57)

### PR DESCRIPTION
**Pull request by teslima0**
* Add Authentification and Connect to Postgre Database

* Final Authentication Implementation

* Testing APi

* handling Images (#54)

* Testing auth APIviews

* Resolving merge conflict

* Resolve merge conflict

Co-authored-by: Wanjiru Murira <wanjirumurira@gmail.com>

# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: CMS/settings.py [hunk 1]
```diff
@@ -87,6 +96,19 @@
 }
 
 
+DATABASES = {
+    'default': {
+        "ENGINE": 'django.db.backends.postgresql',
```
Comment (sangkips) [inline CMS/settings.py:101]: I feel like we are still hardcoding the databse here. What we can do is to create a `.env.sample file` to store this environment variables

Comment (Eugene-Kwaka) [reply CMS/settings.py:101]: Yes. And also include the Django Secret Key and the Allowed Host values in the .env file

**Comment by JimmyTron** [comment]
This pr has become stale

Screenshot_2022-12-19-19-10-00-921_com android chrome

**Pull request closed without merging**